<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ManageTopic</class>
    <widget class="QWidget" name="ManageTopic">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>ManageTopic</string>
        </property>
        <widget class="QPushButton" name="studyButton">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>70</y>
                    <width>211</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>Study</string>
            </property>
        </widget>
        <widget class="QPushButton" name="addDefinitionButton">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>110</y>
                    <width>211</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>Add new defenition</string>
            </property>
        </widget>
        <widget class="QPushButton" name="addQuestionButton">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>150</y>
                    <width>211</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>Add new question</string>
            </property>
        </widget>
        <widget class="QPushButton" name="showContentButton">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>190</y>
                    <width>211</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>Show topic content</string>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>20</y>
                    <width>211</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Topic name</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QPushButton" name="goBackButton">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>230</y>
                    <width>211</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>Go back</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
